samples/bookinfo/src/productpage/Dockerfile

```diff
@@ -29,4 +29,4 @@ COPY requirements.txt /opt/microservices/
 EXPOSE 9080
 WORKDIR /opt/microservices
 RUN python -m unittest discover
-CMD python productpage.py 9080
+CMD touch /opt/microservices/microservice.log && tail -f /opt/microservices/microservice.log & python productpage.py 9080
```

Comment:
missing `&&` after `tail -f /opt/microservices/microservice.log`, note the comment https://github.com/istio/istio/issues/10985#issuecomment-459783710

Reply:
Are we not just backgrounding the `tail ...`? && would wait for tail to complete, which would never?

Reply:
I see, you are right.